keyboards/keyten/kt60_mu/readme.md

```diff
@@ -0,0 +1,42 @@
+# keyten kt60-MU
+
+PCB supporting multiple 60% keyboards.
+Outline is compatible with:
+
+* All common O-Ring mount keyboards
+* Polaris V2 (In theory. At the time of PCB design, this keyboard is in IC stage)
+
+Supports two Type-C connector position:
+
+* Standard left position
+* Central position like Aksara 60
+
+Supports three JST connector position like the following PCBs or keyboards:
+
+* hiney h60
+* Singa Unikorn
+* Polaris V2
+
+![kt60-MU](https://i.imgur.com/72uSWLx.jpg)
```

Comment:
The image here shows a the board supports both 2u and split backspace. We would expect more than the single layout macro defined in `keyboard.json`.

Reply:
I didn't find anything like that in community layouts, so I called it `LAYOUT_60_ansi_tsangan_split_rshift`